### ISSUE TITLE
[EN][Soul Link][Warlock] Soul link is removed on summoning new pet.

### ISSUE THREAD

**Issue by anxanas** [Atlantiss/BugTracker#5365]
Soul link is removed when you summon a new pet with soulburn
http://www.wowhead.com/spell=108415/soul-link
http://www.wowhead.com/spell=74434/soulburn
Proff that it was like this on retail and its actually a bug:
https://www.youtube.com/watch?v=ix3jErY9AoU (6:35 he switches from felguard to felhunter and doesnt lose soul link)

PS: Dont close my topic if anything isnt like it should be according to the BT rules just tell me to change it or do it yourself ty.

**Comment by Kejmil** [comment]
any info about normal summon?

**Comment by anxanas** [comment]
Yes a normal summon will keep it too look at  https://www.youtube.com/watch?v=ix3jErY9AoU (8:39)

**Comment by troublemakerr** [comment]
Bump!

**Comment by blinkmaster** [comment]
bump ninja

**Comment by mrusakovich** [comment]
If u summon the same pet soul link doesn't disappear. In all other cases it disappears...
